messages/subjects.yaml

```diff
@@ -58,7 +54,9 @@ vessel_roll_rate_degps:                 keelson.TimestampedFloat
 vessel_pitch_deg:                       keelson.TimestampedFloat
 vessel_pitch_rate_degps:                keelson.TimestampedFloat
 vessel_yaw_deg:                         keelson.TimestampedFloat
-vessel_yaw_rate_degps:                  keelson.TimestampedFloat
+vessel_yaw_rate_degps:                  keelson.TimestampedFloat # Comes directly from a measurement/calculation ex. gyroscope, IMU & Simulation.
+
+rate_of_turn_degpm:                     keelson.TimestampedFloat # Comes from change over time
```

Comment:
Hmm, Same as the previous comment, arent these just different sources, i.e. `source_id`s in the key expression?